doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -208,12 +209,14 @@ Bluetooth mesh samples
 
   * :ref:`bluetooth_ble_peripheral_lbs_coex` sample, demonstrating how to combine Bluetooth mesh and Bluetooth Low Energy features in a single application.
   * Support for :ref:`zephyr:nrf21540dk_nrf52840`.
-  * :ref:`central_and_peripheral_hrs` sample.
 
 * Updated:
 
-  * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340` in non-secure configuration.
+  * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340`.
```

Comment:
Shouldn't the full sample list be added here?

Reply:
Split in added and updated.